src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -41,6 +46,7 @@ def __init__(
         file_conn_id="",
         chunksize=DEFAULT_CHUNK_SIZE,
         if_exists="replace",
+        ndjson_normalize_sep="_",
```

Comment:
Can you add this to docstring in L24-L33

Reply:
Done.